src/bpkg-build/bpkg-build

```diff
@@ -50,13 +50,18 @@ then
 	printf "Version in debian/control does not match CLI argument\n" >&2
 fi
 
+# Finalise the determined version
 VERSION=${VERSION:-$2}
 
+# Determins where the file is to be written to
```

Comment:
s/Determins/Determines/

Reply:
cba to reopen ssh, gonna make github do the work

```suggestion
# Determines where the file is to be written to
```